src/Pelagos/Bundle/AppBundle/Resources/views/DatasetReview/distribution-info.html.twig

```diff
@@ -52,6 +52,28 @@
     </table>
 </fieldset>
 
+<fieldset>
+    <legend>Cold Storage File Information</legend>
+    <table class="tab-table">
+        <tr>
+            <td>
+                {{ form_label(form.isDatasetFileInColdStorage) }}
+                {{ form_widget(form.isDatasetFileInColdStorage, { 'attr': {'class': 'coldstorage-checkbox'} }) }}
+            </td>
+        </tr>
+        <tr class="row-coldstorage-filesize">
+            <td>
+                {{ form_row(form.datasetFileColdStorageArchiveSize) }}
```

Comment:
Should have units beside it? Easier to understand. They wanted it in bytes, so having scroll wouldn't be that helpful i guess.

Reply:
I put it in but somehow deleted it. See the fix in the DatasetSubmissionType.php (form)

Reply:
👍